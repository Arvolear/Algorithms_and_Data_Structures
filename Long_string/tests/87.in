7017688057375906490000566783404325084091
48338199219475266726161969590865610630