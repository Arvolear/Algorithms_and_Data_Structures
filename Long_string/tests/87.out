7066026256595381756726728752995190694721
6969349858156431223274404813813459473461
339222403377568943912617427891372667484874975707190700907902626173454813487330
145